src/Components/LifecycleChart/LifecycleChart.tsx

```diff
@@ -107,13 +171,24 @@ const LifecycleChart: React.FC<LifecycleChartProps> = ({ lifecycleData }: Lifecy
           right: 50, // Adjusted to accommodate tooltip
           top: 50,
         }}
-        width={chartWidth}
+        // width={chartWidth}
+        // adjust this by number of items
+        height={900}
+        width={900}
       >
         <ChartAxis
           dependentAxis
           showGrid
-          tickFormat={(t: Date) => t.toLocaleDateString('en-US', { year: 'numeric' })}
+          tickFormat={(t: Date) => {
+            return t.toLocaleDateString('en-US', { year: 'numeric' });
```

Comment:
We don't need the return - it was totally valid the way it was before, but you don't have to change it unless the linter complains.

Reply:
I'll remove it